src/functions/usdcommandinfo.md

```diff
@@ -16,9 +16,15 @@ $commandInfo[command_name;property]
 
 #### Properties
 
-* Name - The name of the command
-* Code - The code of the command
-* Aliases - The aliases of the command
+* name - The name of the command
+* code - The code of the command
+* aliases - The aliases of the command
+* executeAt - Returns both if command can be used in `both` dm and guild, `guild` if in guild and `dm` if in dm.
+* whitelist - Returns boolean whether or not the command is whitelisted for blacklisted users.
+* nonPrefixed - Returns boolean whether or not if the author message starts with command name.
+* usage - Returns the usage of the command if set.
+* type - Returns the type of the command. [Here's the list.](../options/commandTypes.md)
+* info - Returns the info of the command if any.
```

Comment:
something like

[custom property]: value added in cmd
is better cause usage is not by default a valid property in aoijs cmd